frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx

```diff
@@ -469,9 +475,15 @@ export class SubscriptionUpdates extends React.Component<
                 {this.state.waitingForUpdate ? (
                   <LoadingInline />
                 ) : (
-                  <Button type="button" isInline onClick={() => approvalModal()} variant="link">
+                  <Button
+                    type="button"
+                    isInline
+                    onClick={() => approvalModal()}
+                    variant={!obj ? 'plain' : 'link'}
+                    isDisabled={!obj}
```

Comment:
Sorry, this check might not be needed. It would be a runtime error if `obj` is not already defined. I assumed we needed the install plan resource.